### ISSUE TITLE
SciML/LTS killing Cxx.jl off

### ISSUE THREAD

**Issue by PallHaraldsson** [SciML/DifferentialEquations.jl#828]
Cxx.jl only supports up to 1.3 currency. So this part is problematic:

https://sciml.ai/news/2021/12/02/lts/index.html
>All packages will be given a minimum version of the LTS
> [..]
>Lastly, we encourage other organizations to adopt a similar strategy.

For SciML, it, or its users may not need C++ (Cxx.jl). I realize fully how you don't need C++, i.e. beat all packages in languages like C++, C and Fortran on speed and functionality. For your users however they might have a C++ dependency, with Cxx or CppWrap, or want to use either, not limited to the latter.

I'm just commenting here, as this is the first such LTS statement I see, but also because of the encouragement. I'm curious can you foresee lack of Cxx a problem for **your** ecosystem? Could you have a lower bound of 1.3+? It might give the wrong impression about them supported. I'm fully happy with:

>bug reports from pre-v1.6 Julia versions which cannot be reproduced on Julia LTS will be closed.

Since 1.3 has a security issue, only 1.6 and 1.7+ fixed, I'm very conflicted about encouraging its use, but still hesitant about forcing others to stop using, by upping version compatibility further.

https://github.com/JuliaInterop/Cxx.jl/issues/490#issuecomment-986638810

**Comment by ChrisRackauckas** [comment]
This isn't an issue for our ecosystem. Cxx.jl isn't used anywhere in the dependency stack. At some point we need to stop supporting unsupported packages, and this sounds like a good time to drop that. People are free to continuing using old versions with manifests that work for v1.3, and the lower bounds of all packages are set to v1.6 so they should never change, but we don't plan on changing anything more around here for those versions.

**Comment by PallHaraldsson** [comment]
Ok understandable, I was fishing for a) if 1.3 could be supported, you might know. And b) of the wording on "encourage other organizations" could be changed. That's a wider discussion, and of course that you are leading.

**Comment by ChrisRackauckas** [comment]
You cannot even download 1.3 using the links from the downloads page anymore, so only old-timers who know what they are doing will even find it. And I definitely encourage other organizations to not spend the hours upon hours to support a version nobody can download.

**Comment by PallHaraldsson** [comment]
Strange it's here: https://julialang.org/downloads/oldreleases/ I could download both versions I tried for Linux and Windows.

**Comment by ChrisRackauckas** [comment]
And it literally says:

>Older Unmaintained Releases
>Old releases are available should you need to use them to run Julia code written for those releases. Note that these are not actively developed nor maintained anymore.

Or in plain English, good luck not my problem.
